src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -86,9 +86,12 @@ case class ClusterImage(tool: ClusterTool,
                         dockerImage: String,
                         timestamp: Instant)
 
+case class ClusterInternalId(value: String) extends ValueObject
+
 // The cluster itself
 // Also the API response for "list clusters" and "get active cluster"
 final case class Cluster(id: Long = 0, // DB AutoInc
+                   internalId: ClusterInternalId,
```

Comment:
I just realized this adds a field to our API json response; will update Swagger.